prime-router/src/test/kotlin/fhirengine/azure/FhirFunctionTests.kt

```diff
@@ -154,8 +154,8 @@ class FhirFunctionTests {
             "\"blobURL\":\"http://azurite:10000/devstoreaccount1/reports/receive%2Fignore.ignore-full-elr%2F" +
             "None-011bb9ab-15c7-4ecd-8fae-0dd21e04d353-20220729171318.hl7\",\"digest\":\"58ffffffaaffffffc22ffffff" +
             "f044ffffff85ffffffd4ffffffc9ffffffceffffff9bffffffe3ffffff8fffffff86ffffff9a5966fffffff6ffffff87fffff" +
-            "fff5bffffffae6015fffffffbffffffdd363037ffffffed51ffffffd3\",\"sender\":\"ignore.ignore-full-elr\"," +
-            "\"blobSubFolderName\":\"ignore.ignore-full-elr\"}"
+            "fff5bffffffae6015fffffffbffffffdd363037ffffffed51ffffffd3\",\"blobSubFolderName\":" +
+            "\"ignore.ignore-full-elr\",\"schemaName\":\"metadata/fhir_transforms/fhir-transform-sample\"}"
```

Comment:
should `fhir-transform-sample` be in a test resources folder instead?

Reply:
It's actually not even used here because of the mocked functions, I just wanted the queue message to more closely resemble a real thing. I'll update the schemaName here to be `someSchema` to make it more clear that it doesn't need to point to a real thing for this test.

Reply:
But yes, conceptually, I agree that if the test relies on a file, that file should be in a `test` folder.